CompilerSource/JDI/src/System/macros.cpp

```diff
@@ -153,7 +153,7 @@ inline void jdip::macro_function::preparse(string val, error_handler *herr)
         value.push_back(mv_chunk(val.c_str(), push_from, i-push_from)); // Push it onto our value
       while (is_useless_macros(val[++i]));
       push_from = i; // Store current position just in case something stupid happens
-      if (!is_letter(val[i])) { herr->error("Expected parameter name following '#' token; `" + val + "'[" + ::toString(i) + "] is not a valid identifier character"); continue; }
+      if (!is_letter(val[i])) { herr->error("Expected parameter name following '#' token; `" + val + "'[" + ::toString((int)i) + "] is not a valid identifier character"); continue; }
```

Comment:
I went and looked through all the branches on master for the file:
https://github.com/enigma-dev/enigma-dev/blob/master/CompilerSource/JDI/src/System/macros.cpp

I can't see anybody modifying except @JoshDreamland since 2012. Even on Harri's GL3.3 branches but I could have swore I seen this change already myself too.